ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -472,4 +476,25 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
             });
         }
     }
+
+    private saveLastVisitedSectionIndex(sectionIndex: number): void {
+      if (this.shouldSaveLastStep && sectionIndex > this.model.sectionIndex) {
+        this.serviceAgent.saveLastVisitedActivitySection(this.studyGuid, this.activityGuid, this.currentSectionIndex);
+      }
+    }
+
+    private getFirstSectionIndex(): void {
+      this.getIsLoaded$()
+        .pipe(
+          filter(value => !!value),
```

Comment:
Or just filter(Boolean).
Even more, I guess `filter(value => value)` would be quite enough as `getIsLoaded$` already `boolean`.
It is not critical, anyway.